languages/javascript/exercises/concept/classes-intro/.docs/introduction.md

```diff
@@ -0,0 +1,63 @@
+In general object-oriented programming, a class is an extensible template for storing associated values and methods.
+
+In JavaScript, a class is a special type of function that can bind values and methods to itself using the `this` keyword.
+
+Initialize classes by using the `class` keyword:
+
+```js
+
+class MyClass() {
+    //...
+}
+
+```
+
+To add a method (a function of a class) to a class, just write the function definition without the `function` keyword in the class body:
+
+```js
+class FunctionDemo {
+  function1() {
+    console.log('Hello classes!')
+  }
+
+  function2() {
+    this.function1()
+
+    console.log('Hello second function!')
+  }
+}
+```
+
+The constructor function is a special kind of function that is run upon class initialization. This is where you can initialize and store class properties:
+
+```js
+class MyConstructorClass {
+  constructor(property1, property2) {
+    this.property1 = property1
+    console.log(property2)
+  }
+}
+```
+
+The `this` keyword is a way to access properties and methods on a class's _prototype_. The prototype is a JavaScript object where all properties and methods of a class are stored.
+
+You can access the `this` keyword from anywhere inside any function:
```

Comment:
I would make it clear that "any function" is "any function" that is on the class